src/index.js

```diff
@@ -0,0 +1,17 @@
+import createHttpConnector from './Connector/CreateHttpConnector';
+import createResourceRepository from './Resource/RepositoryFactory';
+import jsonApiParamsSerializer from './Transformers/JsonApi/paramsSerializer';
+import jsonApiResponseNormalizer from './Transformers/JsonApi/responseNormalizer';
+import jsonApiRequestSerializer from './Transformers/JsonApi/requestSerializer';
+import jsonApiUrlSerializer from './Transformers/JsonApi/urlSerializer';
+import ParameterBag from './Resource/ParameterBag';
+
+const jsonApiTransformers = {
+  pramsSerializer: jsonApiParamsSerializer,
+  responseNormalizer: jsonApiResponseNormalizer,
+  requestSerializer: jsonApiRequestSerializer,
+  urlSerializer: jsonApiUrlSerializer,
+};
+
+export { createHttpConnector, jsonApiTransformers, ParameterBag };
+export default createResourceRepository;
```

Comment:
hmm, misschien deze (createResourceRepository ) ook specifiek exporten?

En dan de volgende default export:

default export: {
  connector: {
    http: createHttpConnector
  }
  resource: {
    create: createResourceRepository
  },
  transformer: {
    jsonApi: jsonApiTransformers
  },
  parameterBag: ParameterBag
}


oid?

Reply:
Is wel een goed idee om die ook specifiek te exporten.
In de default wil ik dan wel een sensible defaults exporteren (een soort batteries included).
Wat je nu voorstelt als default betekent meer dat je dan alle objecten vrij specifiek terugkrijgt. 
Stel dat je voor de default een functie exporteert die op basis van axios een default http connector teruggeeft? Of geen default export...